ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/CommonPartitionEvaluator.java

```diff
@@ -231,14 +280,15 @@ public long getCost() {
       // When rewriting the Position delete file, only the primary key field of the segment file
       // will be read, so only one-tenth of the size is calculated based on the size.
       cost =
-          (rewriteSegmentFileSize + fragmentFileSize) * 4
+          (fragmentFileSize + segmentFileSize + rewriteSegmentFileSize) * 4
```

Comment:
The definition of `segmentFileSize` seems to have changed, we'd better discuss it.

Reply:
Done. Renamed to `undersizedSegmentFileSize`.